force-app/main/default/classes/tasks/BasicTasks.cls

```diff
@@ -24,8 +32,8 @@ public class BasicTasks {
   public static Decimal mathPower(Integer a, Integer n) {
     //write your code below this line
     Decimal result = 0;
-
+    result = Math.pow(a, n);
```

Comment:
The purpose of this task is to practice loops and algorithms construction - see couple of lines above:
// Implement power math operator using while loop, e.g. a = 5, n = 2; then result should be 25

Reply:
Updated